src/app/ui/components/account/account.card.tsx

```diff
@@ -21,7 +21,7 @@ import { PrivateTextLayout } from '@app/components/privacy/private-text.layout';
 import { BasicTooltip } from '../tooltip/basic-tooltip';
 
 const availableBalanceTooltipLabel =
-  'Total balance minus outbound transfers, protected collectibles and uneconomical UTXOs.';
+  'Total balance minus locked amounts, outbound transfers, protected collectibles and uneconomical UTXOs';
```

Comment:
I'm not sure the `.` should have been removed here. I do think we typ include in other tooltips?

Reply:
@markmhendrickson I also think you meant we need to actually deduct first from the new available balance at top?